sdk/communication/communication-verified-recipients/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
+
+## 1.0.0-beta.5 (Unreleased)
```

Comment:
could add some description here marking the first release of the package

Reply:
Here would I just put `## 1.0.0-beta.5 ({todays_date})`? Is that how it typically is?

Reply:
I meant to add some text as well corresponding to this version, describing the package.
Example https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/attestation/attestation/CHANGELOG.md#100-beta1-2021-01-15

Reply:
ok